files/eni-max-pods.txt

```diff
@@ -11,6 +11,11 @@
 # access the instance metadata, VPC DNS, and Time Sync services from the 32nd IP
 # address onwards. If access to these services is needed from all IP addresses
 # on the interface, we recommend using a maximum of 31 IP addresses per interface.
+a1.medium 8
+a1.large 14
+a1.xlarge 20
+a1.2xlarge 20
+a1.4xlarge 39
```

Comment:
Hmm, I think this should be:
```
a1.medium 8
a1.large 29
a1.xlarge 58
a1.2xlarge 58
a1.4xlarge 234
```
See https://github.com/awslabs/amazon-eks-ami/pull/236/files#diff-6ec08afbedd68be4a24ff9c1f9ca668fR14